src/echo.js

```diff
@@ -24,9 +24,9 @@ class PusherConnector {
     /**
      * Create a fresh Pusher connection.
      */
-    static connect(pusherKey, customOptions = {})
+    static connect(pusherKey, { csrfToken, ...customOptions } = {})
```

Comment:
The Spread. http://odetocode.com/blogs/scott/archive/2014/09/02/features-of-es6-part-5-the-spread.aspx